.github/workflows/update_prerelease_version.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import datetime
+import json
+import pathlib
+from typing import Tuple
+
+EXT_ROOT = pathlib.Path(__file__).parent.parent.parent
```

Comment:
"EXT" as in "extension" like the extension itself? Otherwise that can read "extension like a file extension" since you're dealing with file paths.

Reply:
Ah! yes. I will rename it.

Reply:
Actually removed it.